mobile/examples/phi-3/android/app/src/main/java/ai/onnxruntime/genai/demo/MainActivity.java

```diff
@@ -41,7 +41,10 @@ public class MainActivity extends AppCompatActivity implements Consumer<String>
     private TextView generatedTV;
     private TextView promptTV;
     private TextView progressText;
+    private ImageButton settingsButton;
     private static final String TAG = "genai.demo.MainActivity";
+    private int maxLength = 100;
+    private int lengthPenalty = 1000;
```

Comment:
According to [this](https://onnxruntime.ai/docs/genai/reference/config.html) doco the default for length penalty is 1, with values < 1 encouraging shorter responses and > 1 longer. Is that doco out of date? Or if not, should this be a float and start at 1.0?